src/Layouts/Partials/FillItem.vala

```diff
@@ -191,20 +201,38 @@ public class Akira.Layouts.Partials.FillItem : Gtk.Grid {
         color_popover.add (color_picker);
 
         attach (fill_chooser, 0, 0, 1, 1);
-        attach (hidden_button, 1, 0, 1, 1);
-        attach (delete_button, 2, 0, 1, 1);
+        attach (eye_dropper_button, 1, 0, 1, 1);
+        attach (hidden_button, 2, 0, 1, 1);
+        attach (delete_button, 3, 0, 1, 1);
 
         set_color_chooser_color ();
         set_button_color ();
     }
 
     private void create_event_bindings () {
+        eye_dropper_button.clicked.connect (on_eye_dropper_click);
```

Comment:
Instead of triggering this from within the fill item, I think we should have a globally accessible action so users can quickly copy a color with an handy shortcut.
The shortcut should be `Alt+C`, and the action should check if not items are currently selected.
Once the color is picked, the method should loop through all the currently selected items and apply the same color (since we're working on multiselect, it's better to have things in place right at the start).

Reply:
The color picker must work with every part that contains colors, such as fills and borders, if we add it as a globally accessible action how do we know where it should be applied, borders or fills?

Reply:
By default it should interact with fills, but if the user holds the SHIFT key down when selecting a color, it'll be applied to the border.

Reply:
@Alecaddd I'm having a problem setting the color after picking via global action, it is not being reactive.
![Screen record from 2020-08-31 18 57 31](https://user-images.githubusercontent.com/8999039/91745944-19f3ab80-ebbc-11ea-88f2-56370837e592.gif)

Reply:
I need to look at the code of the action to check potential issues.

Reply:
I've pushed the code with the global action, please advise. thanks.